syntax = "proto3";
package calculatie;


message CalculatePriceRequest {
  string building_specification_id = 1; 
  string name = 2; 
  string unit = 3;
  float quantity = 4;             
  float price_per_unit = 5;   
}

message CalculatePriceResponse {
  string building_specification_id = 1;
  float total_price = 2;
}

service CalculationService {
  rpc CalculateProject (stream CalculatePriceRequest) returns (stream CalculatePriceResponse);
}
